gluoncv/model_zoo/yolo/yolo3.py

```diff
@@ -543,7 +625,7 @@ def yolo3_darknet53_coco(pretrained_base=True, pretrained=False,
         'darknet53', stages, [512, 256, 128], anchors, strides, classes, 'coco',
         pretrained=pretrained, norm_layer=norm_layer, norm_kwargs=norm_kwargs, **kwargs)
 
-def yolo3_darknet53_custom(classes, transfer=None, pretrained_base=True, pretrained=False,
```

Comment:
Is here breaking the API? Seems if user explicitly set `pretrained=False`, then in L665 `pretrained=True` will conflict `**kwargs`.

Reply:
pretrained argument should not be True for custom models. add a warning and ignore anyway.